src/coredata.cs

```diff
@@ -1818,8 +2027,24 @@ partial interface NSPersistentStoreCoordinator
 		[NoWatch][NoTV]
 		[Since (7,0), Mavericks]
 		[Field ("NSPersistentStoreUbiquitousContainerIdentifierKey")]
+		[Obsolete ("Use UbiquitousContainerIdentifierKey instead.")]
 		NSString eUbiquitousContainerIdentifierKey { get; }
 
+		[NoWatch][NoTV]
+		[Since (7,0), Mavericks]
+		[Field ("NSPersistentStoreUbiquitousContainerIdentifierKey")]
+		NSString UbiquitousContainerIdentifierKey { get; }
```

Comment:
In a separate pr you can add a check that public fields (all members in fact) should start with an uppercase letter _unless_ it has an obsolete attribute